apps/site/src/app/login/page.tsx

```diff
@@ -0,0 +1,98 @@
+"use client";
+import { useRouter } from "next/navigation";
+
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useForm } from "react-hook-form";
+import * as z from "zod";
+import {
+	Form,
+	FormItem,
+	FormControl,
+	FormDescription,
+	FormLabel,
+	FormField,
+	FormMessage,
+} from "@/components/ui/form";
+import { Input } from "@/components/ui/input";
+import { Button } from "@/components/ui/button";
+import { Separator } from "@/components/ui/separator";
+
+export const revalidate = 60;
+
+const LOGIN_PATH = "/api/user/login";
+
+const formSchema = z.object({
+	email: z
+		.string()
+		.email({ message: "Sorry, that email address is invalid." })
+		.refine((email) => email.endsWith(".edu"), {
```

Comment:
@taesungh @samderanova Are we okay removing .edu validation to allow high school students to sign up?

Probably also need to add a message saying to use school email if you have one